internal/cmd/autostart.go

```diff
@@ -0,0 +1,99 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/xerrors"
+
+	"cdr.dev/coder-cli/coder-sdk"
+	"cdr.dev/coder-cli/internal/x/xcobra"
+	"cdr.dev/coder-cli/pkg/clog"
+)
+
+func autostartCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "autostart",
+		Short: "Interact with Coder autostart settings",
+		Long:  "Perform operations on the Coder autostart settings owned by the active user.",
+	}
+
+	cmd.AddCommand(getAutostartCmd(), setAutostartCmd())
+
+	return cmd
+}
+
+func getAutostartCmd() *cobra.Command {
+	var outputFmt string
+
+	cmd := &cobra.Command{
+		Use:   "get",
+		Short: "get autostart settings owned by the active user",
+		Long:  "Get Coder autostart settings owned by the active user.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+			client, err := newClient(ctx)
+			if err != nil {
+				return err
+			}
+
+			as, err := client.MyAutoStart(ctx)
+			if err != nil {
+				return err
+			}
+
+			switch outputFmt {
+			case humanOutput:
+				fmt.Printf("AutoStart Time:\t%v\n", as.AutoStart)
```

Comment:
You _could_ use `tablewriter` here if you wanted to.

Reply:
https://github.com/cdr/coder-cli/pull/236/commits/bcb983046de2720621de4ad21f1c8da05305c0f7 I think that's the intended way to do it here (unsure)?

Reply:
was thinking you could make a pseudo slice with one element, but it doesn't matter either way. whatever you think is cleanest.